main/settings.py

```diff
@@ -580,14 +580,21 @@ def get_all_config_keys():
     "AI_MIT_SYLLABUS_URL",
     "https://api.learn.mit.edu/api/v0/vector_content_files_search/",
 )
+AI_MIT_VIDEO_METADATA_URL = get_string(
+    "AI_MIT_VIDEO_METADATA_URL",
+    "https://api.rc.learn.mit.edu/api/v0/vector_content_files_search/",
```

Comment:
Other defaults are for prod, might be best to make this one as well for consistency (or change the others to RC) and make appropriate changes in the `ol-infrstructure` repo (example here - https://github.com/mitodl/ol-infrastructure/pull/3007/files)